docs/StardustDocs/topics/gettingStarted/gettingStarted.md

```diff
@@ -1,87 +0,0 @@
-[//]: # (title: Get started with Kotlin DataFrame)
```

Comment:
You renamed the files, does that mean the URLs have changed as well?

Reply:
yes